components/cart/cart.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/7389171962.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://vite-ui.netlify.app/css/component.css">
+    <link rel="stylesheet" href="/css/landing.css">
+    <link rel="stylesheet" href="/css/cart.css">
+    <title>Document</title>
```

Comment:
We can change this title from Document to My Cart.

Reply:
sure, thanks